sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -4,18 +4,572 @@
 #include "azure/core/http/http.hpp"
 #include "azure/core/http/winhttp/win_http_client.hpp"
 
+#include <Windows.h>
+#include <algorithm>
+#include <iostream>
 #include <string>
+#include <winhttp.h>
 
 using namespace Azure::Core::Http;
 
-WinHttpTansport::WinHttpTansport() : HttpTransport() {}
+// WinHttpTransport::WinHttpTransport() : HttpTransport() {}
+//
+// WinHttpTransport::~WinHttpTransport() {}
 
-WinHttpTansport::~WinHttpTansport() {}
+std::wstring HttpMethodToWideString(HttpMethod method)
```

Comment:
There's no reason to use wstring for the httpmethod as they will always be ansi.

Reply:
This is wstring not because the text is ANSI, but because the WinHTTP APIs need wstring. I don't want to encode from std::string to wstring at the API call, which is why I made these ASCII-only method strings wstring.

I could keep them string and do a re-interpret cast here though, but with the current approach that isn't needed.